### ISSUE TITLE
 double free or corruption (!prev) and tensorboard show nothing

### ISSUE THREAD

**Issue by wxy920801** [tensorflow/tensorflow#7839]
I'm new to use tensorflow.
And I want to  learn how to use tensorboard. 
So I run the demo code mnist_with_summaries.py, 
But in the end it shows  
 
*** Error in `/home/wxy/anaconda2/envs/tensorflow/bin/python': double free or corruption (!prev): 0x0000000002166320 ***

And I input tensorboard --logdir=/tmp/mnist/logs and open  http://0.0.0.0:6006/ it shows nothing.
so I follow the guide to input 

/tmp/mnist/logs | grep tfevents

then

/tmp/mnist/logs/test/events.out.tfevents.1487915616.wxy
/tmp/mnist/logs/train/events.out.tfevents.1487915615.wxy

input

tensorboard --inspect --logdir=/tmp/mnist/logs

then 

Processing event files... (this can take a few minutes)
======================================================================

Found event files in:
/tmp/mnist/logs/test
/tmp/mnist/logs/train

These tags are in /tmp/mnist/logs/test:
audio -
histograms
   layer1/Wx_plus_b/pre_activations
   layer1/activations
   layer1/biases/summaries/histogram
   layer1/weights/summaries/histogram
   layer2/Wx_plus_b/pre_activations
   layer2/activations
   layer2/biases/summaries/histogram
   layer2/weights/summaries/histogram
images
   input_reshape/input/image/0
   input_reshape/input/image/1
   input_reshape/input/image/2
   input_reshape/input/image/3
   input_reshape/input/image/4
   input_reshape/input/image/5
   input_reshape/input/image/6
   input_reshape/input/image/7
   input_reshape/input/image/8
   input_reshape/input/image/9
scalars
   accuracy_1
   cross_entropy_1
   dropout/dropout_keep_probability
   layer1/biases/summaries/max
   layer1/biases/summaries/mean
   layer1/biases/summaries/min
   layer1/biases/summaries/stddev_1
   layer1/weights/summaries/max
   layer1/weights/summaries/mean
   layer1/weights/summaries/min
   layer1/weights/summaries/stddev_1
   layer2/biases/summaries/max
   layer2/biases/summaries/mean
   layer2/biases/summaries/min
   layer2/biases/summaries/stddev_1
   layer2/weights/summaries/max
   layer2/weights/summaries/mean
   layer2/weights/summaries/min
   layer2/weights/summaries/stddev_1
======================================================================

Event statistics for /tmp/mnist/logs/test:
audio -
graph -
histograms
   first_step           0
   last_step            990
   max_step             990
   min_step             0
   num_steps            100
   outoforder_steps     []
images
   first_step           0
   last_step            990
   max_step             990
   min_step             0
   num_steps            100
   outoforder_steps     []
scalars
   first_step           0
   last_step            990
   max_step             990
   min_step             0
   num_steps            100
   outoforder_steps     []
sessionlog:checkpoint -
sessionlog:start -
sessionlog:stop -
======================================================================

These tags are in /tmp/mnist/logs/train:
audio -
histograms
   layer1/Wx_plus_b/pre_activations
   layer1/activations
   layer1/biases/summaries/histogram
   layer1/weights/summaries/histogram
   layer2/Wx_plus_b/pre_activations
   layer2/activations
   layer2/biases/summaries/histogram
   layer2/weights/summaries/histogram
images
   input_reshape/input/image/0
   input_reshape/input/image/1
   input_reshape/input/image/2
   input_reshape/input/image/3
   input_reshape/input/image/4
   input_reshape/input/image/5
   input_reshape/input/image/6
   input_reshape/input/image/7
   input_reshape/input/image/8
   input_reshape/input/image/9
scalars
   accuracy_1
   cross_entropy_1
   dropout/dropout_keep_probability
   layer1/biases/summaries/max
   layer1/biases/summaries/mean
   layer1/biases/summaries/min
   layer1/biases/summaries/stddev_1
   layer1/weights/summaries/max
   layer1/weights/summaries/mean
   layer1/weights/summaries/min
   layer1/weights/summaries/stddev_1
   layer2/biases/summaries/max
   layer2/biases/summaries/mean
   layer2/biases/summaries/min
   layer2/biases/summaries/stddev_1
   layer2/weights/summaries/max
   layer2/weights/summaries/mean
   layer2/weights/summaries/min
   layer2/weights/summaries/stddev_1
======================================================================

Event statistics for /tmp/mnist/logs/train:
audio -
graph
   first_step           0
   last_step            0
   max_step             0
   min_step             0
   num_steps            1
   outoforder_steps     []
histograms
   first_step           1
   last_step            999
   max_step             999
   min_step             1
   num_steps            900
   outoforder_steps     []
images
   first_step           1
   last_step            999
   max_step             999

Can you tell me what's the problem? 
How could I deal with it?
So I want some people to help me.
Thank you so much!

**Comment by poxvoculi** [comment]
This forum is for bug reports.  For use issues, please post to stackoverflow.   Since TensorFlow recently relased version 1.0 please make sure you're running the latest version.  If you suspect a bug, please report all the information requested in the issues template.

**Comment by yaroslavvb** [comment]
@wxy920801 can you see if suggestions here help? https://github.com/tensorflow/tensorflow/issues/6968
cc @jhseu

**Comment by wxy920801** [comment]
@yaroslavvb Thank you！！
I can't believe that has solve my problem!

**Comment by wxy920801** [comment]
@yaroslavvb 
Thank you!!
I can't believe that solves my problem!

**Comment by yaroslavvb** [comment]
@wxy920801 meaning you got tensorboard running by setting LD_PRELOAD ?

**Comment by chenliu0831** [comment]
Confirm `LD_PRELOAD="/usr/lib/libtcmalloc_minimal.so.4" tensorboard --logdir ...` fixed it for me
